api/src/paths/public/projects.ts

```diff
@@ -62,24 +67,25 @@ export function getPublicProjectsList(): RequestHandler {
 
       await connection.open();
 
-      const getProjectListResponse = await connection.query(
-        getProjectListSQLStatement.text,
-        getProjectListSQLStatement.values
-      );
+      const response = await connection.query(getProjectListSQLStatement.text, getProjectListSQLStatement.values);
 
       await connection.commit();
 
-      let rows: any[] = [];
-
-      if (getProjectListResponse && getProjectListResponse.rows) {
-        rows = getProjectListResponse.rows;
+      if (!response || !response.rows) {
```

Comment:
nit: can remove the `!response` check.  

I had the incorrect type definition before (in my db wrapper) which indicated that the response could be null, but in reality the underlying library will never return null.